index.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script defer src="./assets/js/app.js"></script>
+    <title>Awsome Books App</title>
+</head>
+<body>
```

Comment:
- [ ] I'm also trying to open your app with a live server, but I can't add or remove books. Please make sure the add and remove book functionality is implemented before submitting for another precedent. 🙏🏻

Reply:
@BenMukebo Hi, there was a problem with the function but we have fixed it now can you please rereview the project because we want to start the next project tomorrow if we lose one more review it will cost us a lot as there won't be much time left for the tomorrow's task.

Reply:
Hi @caasperr, please request the second review,